packages/sdks-tests/src/tests/blocks.spec.ts

```diff
@@ -61,10 +64,31 @@ test.describe('Blocks', () => {
    * We are temporarily skipping this test because it relies on network requests.
    * TO-DO: re-enable it once we have a way to mock network requests.
    */
-  test.skip('Image', async ({ page }) => {
+  test('Image', async ({ page, packageName }) => {
```

Comment:
Nice!! Can you remove the comment above that says this is temporarily disabled?

Reply:
sure!